packages/console-client/src/kiota-client/api/extensibility/tenants/item/extensions/index.ts

```diff
@@ -375,7 +635,8 @@ export function serializeExtensions500Error(writer: SerializationWriter, extensi
  * @param writer Serialization writer to use to serialize this model
  */
 export function serializeExtensionsPutRequestBody(writer: SerializationWriter, extensionsPutRequestBody: Partial<ExtensionsPutRequestBody> | undefined = {}) : void {
-    writer.writeCollectionOfPrimitiveValues<string>("contexts", extensionsPutRequestBody.contexts);
+    if(extensionsPutRequestBody.contexts)
```

Comment:
this change is related to https://github.com/microsoft/kiota-typescript/issues/1276